mixitup/csv_json.py

```diff
@@ -0,0 +1,20 @@
+import csv, ujson as json, pprint
```

Comment:
Don't mix a Python file in with front-end web code. Make a `scripts` directory in the root of the repo for Python files.